src/Stripe.net/Entities/Reporting/ReportRun.cs

```diff
@@ -0,0 +1,39 @@
+namespace Stripe.Reporting
+{
+    using System;
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+    using Stripe.Infrastructure;
+
+    public class ReportRun : StripeEntityWithId
+    {
+        [JsonProperty("object")]
+        public string Object { get; set; }
+
+        [JsonProperty("created")]
+        [JsonConverter(typeof(StripeDateTimeConverter))]
+        public DateTime Created { get; set; }
+
+        [JsonProperty("error")]
+        public string Error { get; set; }
+
+        [JsonProperty("livemode")]
+        public bool LiveMode { get; set; }
```

Comment:
Ugh, I don't like that the capitalization doesn't match the snake_case -> CamelCase rule here, but it's consistent with every other resource, so let's keep it as is.

Reply:
yup I think that every time I do a new resource. Adding to the list of things to change in the next major revision